prog_test.go

```diff
@@ -91,7 +91,7 @@ func TestProgramRun(t *testing.T) {
 }
 
 func TestProgramRunWithOptions(t *testing.T) {
-	testutils.SkipOnOldKernel(t, "5.15", "XDP ctx_in/ctx_out")
+	testutils.SkipOnOldKernel(t, "5.10", "RawTracepoint test run")
```

Comment:
This needs to go in TestProgramRunRawTracepoint :)

Reply:
yes yes I just realized I modified the wrong one ahah

Reply:
https://github.com/cilium/ebpf/compare/35b0c1b6cedb6c5f92c0eb04e2ee5eaae955d853..f810c0a9dec0414378dbfb63cdb6e4188fa62619